### ISSUE TITLE
Gradle 4.4 support

### ISSUE THREAD

**Issue by fkorotkov** [gogradle/gogradle#159]
After upgrading to Gradle 4.4 `test` task is failing with:

```
Caused by: java.lang.NoClassDefFoundError: org/gradle/api/internal/tasks/testing/junit/report/DefaultTestReport
        at com.github.blindpirate.gogradle.unsafe.GradleInternalAPI.renderTestReport(GradleInternalAPI.java:55)
        at com.github.blindpirate.gogradle.task.go.GoTest.generateTestReport(GoTest.java:152)
        at com.github.blindpirate.gogradle.task.go.GoTest.run(GoTest.java:118)
```

**Comment by blindpirate** [comment]
Oh, thanks for reporting. Gogradle uses some internal Gradle service to render test report, which may result in this error. I'll fix it soon.

**Comment by fkorotkov** [comment]
Sure thing. Just wanted to let you know. 😅  If I'll have time over the weekend I'll take a look at the issue.

**Comment by blindpirate** [comment]
Fixed by `0.8.0`. Any feedbacks are welcomed.
